src/test/ui/existential_types/existential-types-with-no-traits.stderr

```diff
@@ -0,0 +1,14 @@
+error: at least one trait must be specified
+  --> $DIR/existential-types-with-no-traits.rs:3:1
+   |
+LL | existential type Foo: 'static;
```

Comment:
can you limit the span to just the bounds? Maybe iterate over the bounds and collect all the spans into one `Vec` and use https://doc.rust-lang.org/nightly/nightly-rustc/syntax/source_map/struct.MultiSpan.html#method.from_spans to create a multispan. Then use that multispan to report the error.

Reply:
Okay, I used multispan instead.